### PR TITLE
Fix code block showing backticks

### LINKED ISSUE #8563: Backtics ``` are showing
&#96;&#96;&#96;console
dotnet dev-certs https --trust
&#96;&#96;&#96;

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca6139d4-022c-395a-0abf-539660550df1
* Version Independent ID: 3dd23f8a-300e-3f78-1dc5-8b759aabdccc
* Content: [Get started with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-2.1&tabs=windows)
* Content Source: [aspnetcore/getting-started/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/getting-started/index.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by pyrox18**
Indenting the code block with 4 spaces causes the Markdown parser to think that the lines with the backticks are also part of the code block, [as per the Markdown specifications](https://daringfireball.net/projects/markdown/syntax#precode). I reduced the indentation to 2 spaces to get around this problem.

Fixes #8563 

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-2.1&branch=pr-en-us-8772)

### CODE REVIEW

**Comment by guardrex** [comment]
Oh ... and the CLA note just popped up. @pyrox18, can you sign the CLA :point_up:.

**Comment by guardrex** [comment]
btw @scottaddie, we still have this problem where the CLA notice appears **after** we look/update the PR. That didn't show up until after my commit, so it resulted in a violation of our policy.

File: aspnetcore/getting-started/index.md [hunk 1]
```diff
@@ -15,57 +15,56 @@ uid: getting-started
...
+  ```
 
-   The preceding command displays the following message:
+  The preceding command displays the following message:
 
-   *Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:*
-   `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`
-   *This command might prompt you for your password to install the certificate on the system keychain.
-   Password:*
+  * Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:* `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`.
```
Comment (guardrex) [inline aspnetcore/getting-started/index.md]: This line ... prob others ... could use a bit of UE work, too. I was only looking at indentation on my pass. If you want me to open a UE issue for this and work it, let me know.

Comment (pyrox18) [reply aspnetcore/getting-started/index.md]: Sure, I think I have some time over the next few days. Just need some pointers on how I should modify it so I have some guidelines to follow, if possible.

Also, did you intend to put a space between the `*` and `Trusting`? It's now rendering as an unordered list.

Comment (guardrex) [reply aspnetcore/getting-started/index.md]: Oh ....... yeah. Hold on a sec. I'll fix it.

**Comment by guardrex** [comment]
@pyrox18 - yes, sure. Our guidelines are https://docs.microsoft.com/en-us/style-guide/welcome/ and *The Chicago Manual of Style*. I see several little things that could/should be addressed. We can merge this PR now and then you can open an issue to do a UE ("user experience") pass on a new PR.

**Comment by scottaddie** [comment]
Thank you for fixing this, @pyrox18! The changes will go live within the next 24 hours.

**Pull request merged**